PayPal/src/main/java/com/braintreepayments/api/paypal/AmountBreakdown.kt

```diff
@@ -0,0 +1,49 @@
+package com.braintreepayments.api.paypal
+
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
+
+/**
+ * A recurring billing amount breakdown.
+ *
+ * This object is only used in `PayPalCheckoutRequest` to customize how the transaction amount is
+ * broken down. If provided, `itemTotal` becomes required. Some fields are conditionally required or
+ * not accepted based depending on the checkout flow (e.g., one-time vs subscription).
+ *
+ * @property itemTotal - Total amount of the items before any taxes or discounts.
+ *                       Required when `amountBreakdown` is provided.
```

Comment:
```suggestion
```
We don't need to specify that since this property is required on the `amountBreakdown` object.